packages/designsystem/src/components/IdentityCard/IdentityCard.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="flex flex-row w-full py-1 shadow h-12">
+  <div class="flex flex-row w-full py-1 shadow h-12 rounded">
```

Comment:
I don't rememeber, is the `rounded` class related to a CSS variable, to fix the radius ?

Reply:
- `rounded` is form tailwind
- `lck-rounded`: depends from a CSS variable (we still in discussed see https://github.com/locokit/locokit/issues/224)

Reply:
Mmm, ok, thanks for the reminder... You know me well... :'(